buddyboss/buddyboss-platform
# PROD-1824 - Reply button label on Email does not fit when translated to a longer string

### All Submissions:






### Changes proposed in this Pull Request:



Fixes PROD-1824.

### How to test the changes in this Pull Request:

1. Change language or force translate the "Reply" string for the email notification with longer strings when receiving new messages
2. Test a sample user and go to his Email to check the notification
3. See it looks fine.

### Proof Screenshots or Video

https://www.loom.com/share/d8b20967d1714f42aaaa43b768cfc67b

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
